waspc/cli/src/Wasp/Cli/Command/CreateNewProject/StarterTemplates.hs

```diff
@@ -106,55 +108,62 @@ openSaasStarterTemplate =
       "Everything a SaaS needs! Comes with Auth, ChatGPT API, Tailwind, Stripe payments and more."
         <> " Check out https://opensaas.sh/ for more details."
     )
-    (\projectDirName ->
-      unlines [
-        {- ORMOLU_DISABLE -}
-                                        "To run your new app, do:",
-        Term.applyStyles [Term.Bold] $  "    cd " <> projectDirName FP.</> "app",
-        Term.applyStyles [Term.Bold]    "    wasp db start",
-                                        "Then open new terminal window/tab in that same dir and do:",
-        Term.applyStyles [Term.Bold]    "    wasp db migrate-dev",
-        Term.applyStyles [Term.Bold]    "    cp .env.server.example .env.server",
-        Term.applyStyles [Term.Bold]    "    wasp start",
-                                        "",
-        Term.applyStyles [Term.Bold]    "Check the README for additional guidance and link to docs!"
-        {- ORMOLU_ENABLE -}
-      ]
+    ( \projectDirName ->
+        unlines
+          [ styleText $ "To run your new app, do:",
+            styleCode $ "    cd " <> projectDirName FP.</> "app",
+            styleCode $ "    wasp db start",
+            styleText $ "Then open new terminal window/tab in that same dir and do:",
+            styleCode $ "    wasp db migrate-dev",
+            styleCode $ "    cp .env.server.example .env.server",
+            styleCode $ "    wasp start",
+            styleText $ "",
+            styleText $ "Check the README for additional guidance and link to docs!"
+          ]
     )
 
+{- HLINT ignore todoTsStarterTemplate "Redundant $" -}
+
 todoTsStarterTemplate :: StarterTemplate
 todoTsStarterTemplate =
   simpleGhRepoTemplate
     ("starters", [reldir|todo-ts|])
     ( "todo-ts",
       "Simple but well-rounded Wasp app implemented with Typescript & full-stack type safety."
     )
-    (\projectDirName ->
-      unlines [
-        {- ORMOLU_DISABLE -}
-                                        "To run your new app, do:",
-        Term.applyStyles [Term.Bold] $  "    cd " ++ projectDirName,
-        Term.applyStyles [Term.Bold]    "    wasp db migrate-dev",
-        Term.applyStyles [Term.Bold]    "    wasp start",
-                                        "",
-        Term.applyStyles [Term.Bold]    "Check the README for additional guidance!"
-        {- ORMOLU_ENABLE -}
-      ]
+    ( \projectDirName ->
+        unlines
+          [ styleText $ "To run your new app, do:",
+            styleCode $ "    cd " ++ projectDirName,
+            styleCode $ "    wasp db migrate-dev",
+            styleCode $ "    wasp start",
+            styleText $ "",
+            styleText $ "Check the README for additional guidance!"
+          ]
     )
 
+{- Functions for styling instructions. They are on purpose of same length, for nicer formatting. -}
```

Comment:
What is the same length, the function name or the function output?

Reply:
Function names! I improved the comment.
I played with with telling Ormolu to ignore these parts, but yo can tell it that only for the whole function, and then you loose formatting for the whole function which was annoying. So I went with this at the end.